Author: Sergei Golovan
Description: Patch fixes a typo in -- switch processing.
Last-Modified:  Wed, 21 Mar 2018 11:04:36 +0300

--- a/generic/bltSwitch.c
+++ b/generic/bltSwitch.c
@@ -322,7 +322,7 @@
     for (count = 0; count < argc; count++) {
 	arg = argv[count];
 	if (flags & BLT_SWITCH_OBJV_PARTIAL) {
-	    if ((arg[0] != '-') || ((arg[1] == '-') && (argv[2] == '\0'))) {
+	    if ((arg[0] != '-') || ((arg[1] == '-') && (arg[2] == '\0'))) {
 		/* 
 		 * If the argument doesn't start with a '-' (not a switch)
 		 * or is '--', stop processing and return the number of
